test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -65,6 +65,16 @@ var _ = framework.KubeDescribe("[HPA] Horizontal pod autoscaling (scale resource
 		})
 	})
 
+	framework.KubeDescribe("[Serial] [Slow] StatefulSet", func() {
```

Comment:
@janetkuo @enisoc It looks like e2e tests for scale subresource should be based on HPA's existing framework. By referring [this](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/horizontal-pod-autoscaler.md#horizontalpodautoscaler-object), `HorizontalPodAutoscalerSpec` type has a type named `SubresourceReference` referring to the scale subresource.